### PR TITLE
Content changes

### PR DESCRIPTION
**Pull request by angusfretwell**

### CODE REVIEW

File: src/locales/en.js [hunk 1]
```diff
@@ -72,20 +72,20 @@ export default {
     network: {
       label: 'Digging deeper',
       heading: 'Network Consensus',
-      body: 'By centralising large amounts of power into a handful of mining pools, mining represents one of the biggest flaws to the adoption of crypto assets. It\'s inefficient too, costing millions of dollars each month and processing just seven transactions a second. Skycoin uses a new consensus algorithm, Network Consensus which requires no mining.',
+      body: 'The oligarchical concentration of power into a handful of mining pools is the greatest threat to the security of existing crypto assets. The power usage of existing consensus algorithm is also unsubstainable and Bitcoin\'s PoW algorithm uses enough electricity to power 50,000 homes, merely to process 6 transactions a second. This cost of millions of dollars per month is passed to the users. Skycoin uses a new consensus algorithm, Network Consensus which requires no mining.',
```
Comment (angusfretwell) [inline src/locales/en.js]: >oligarchical concentration of power

As Enda mentioned, I think we might lose some people with this kind of language. It sounds a bit "tin-foil hat" or something... I think we could communicate the same sentiment with different language?

File: src/locales/en.js [hunk 2]
```diff
@@ -72,20 +72,20 @@ export default {
...
+      body: 'As coins aren’t mined, Skycoin distribution is unlike that of mined crypto currencies. Instead of coins going to the miners, new Skycoin are distributed to contributors, developers and investors. Skycoin\'s distribution is designed to get coins in the hands of the users and people building out the ecosystem, instead of giving all the new coins out to miners.',
       link: 'Learn about distribution',
     },
     roadmap: {
       heading: 'Roadmap',
-      blurb: 'Development can be followed and contributed to on GitHub. We will be opening bounties for bug fixes, design and development in the coming months.',
+      blurb: 'Skycoin development can be followed on GitHub. We will be opening bounties for bug fixes, design and development in the coming months.',
       roadmapLink: 'See the full roadmap',
-      community: 'Join Skycoin on Telegram to start contributing, and keep up-to-date with the latest developments.',
+      community: 'Join the Skycoin community on Telegram.',
```
Comment (angusfretwell) [inline src/locales/en.js]: I think it would be good to highlight _why_ people would join the community; what will they get from it?

Comment (gz-c) [inline src/locales/en.js]: The general crypto community understands what it is for

File: src/locales/en.js (hunk 1 above)
Comment (gz-c) [inline src/locales/en.js]: it reads the same without the word "oligarchical"

File: src/locales/en.js (hunk 2 above)
Comment (SkyAudit) [inline src/locales/en.js]: To keep up to date with Skycoin’s latest developments or to start contributing, join Skycoin on Telegram.

Comment (SkyAudit) [inline src/locales/en.js]: less salesy pitch

**Pull request merged**